.github/workflows/codspeed.yml

```diff
@@ -14,16 +14,16 @@ env:
 
 jobs:
   benchmarks:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
```

Comment:
why not keep it at 22.04 ?

Reply:
I was trying to future proof to avoid having to keep coming back to it.

Reply:
switched all to Ubuntu-22.04 for consistency